abci/client/grpc_client.go

```diff
@@ -65,7 +65,7 @@ RETRY_LOOP:
 
 	ENSURE_CONNECTED:
 		for {
-			_, err := client.Echo(context.Background(), &types.RequestEcho{Message: "hello"}, grpc.FailFast(true))
+			_, err := client.Echo(context.Background(), &types.RequestEcho{Message: "hello"}, grpc.WaitForReady(true))
```

Comment:
Fail fast will be deprecated: https://godoc.org/google.golang.org/grpc#FailFast, the suggestion is to use `WaitForReady`